great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -98,35 +109,40 @@ def __init__(
     ) -> None:
         """
         Args:
-            name: the name of this parameter -- this is user-specified parameter name (from configuration);
-            it is not the fully-qualified parameter name; a fully-qualified parameter name must start with "$parameter."
-            and may contain one or more subsequent parts (e.g., "$parameter.<my_param_from_config>.<metric_name>").
+            name: the name of this parameter -- this is user-specified parameter name (from configuration); it is not
+                the fully-qualified parameter name; a fully-qualified parameter name must start with "$parameter."
+                and may contain one or more subsequent parts (e.g., "$parameter.<my_param_from_config>.<metric_name>").
             metric_name: the name of a metric used in MetricConfiguration (must be a supported and registered metric)
             metric_domain_kwargs: used in MetricConfiguration
             metric_value_kwargs: used in MetricConfiguration
             enforce_numeric_metric: used in MetricConfiguration to insure that metric computations return numeric values
             replace_nan_with_zero: if False, then if the computed metric gives NaN, then exception is raised; otherwise,
-            if True (default), then if the computed metric gives NaN, then it is converted to the 0.0 (float) value.
+                if True (default), then if the computed metric gives NaN, then it is converted to the 0.0 (float) value.
```

Comment:
@NathanFarmer Is this just the indentation?

Reply:
@alexsherstinsky I find it very difficult to read without the indentation.